dnsimple/exceptions.py

```diff
@@ -4,16 +4,30 @@ class DNSimpleException(Exception):
 
     :param message: str
         The raw response from the server
-    :param errors: array[str]
-        An array of error messages
+    :param attribute_errors: dict
+        A dict of attribute error
+    :param http_response: requests.Response
+        HTTP response object
     """
 
-    def __init__(self, message=None, errors=None):
+    def __init__(self, message=None, attribute_errors=None, http_response=None):
         self.message = message
-        self.errors = errors
-        super().__init__(self.message)
+        self.attribute_errors = attribute_errors
```

Comment:
Do you think it would be worth adding this without introducing breaking changes to the constructor signature and attributes of the class?

Reply:
The class is only used internally, and in a single place, so I don't have any concerns with modifying its signature and constructor method. Any particular concern that you might have (I could be missing something)?

Reply:
Nope, I'm good :) I only mentioned just in case